src/networks.json

```diff
@@ -414,7 +414,8 @@
       "url": "https://www.oklink.com/manta"
     },
     "start": 332890,
-    "logo": "ipfs://bafkreibbrwgwdcfh755dqh4ndta6tja77qbaa36bfv4pstub6prhhore7a"
+    "logo": "ipfs://bafkreibbrwgwdcfh755dqh4ndta6tja77qbaa36bfv4pstub6prhhore7a",
+    "subscription": "Feb 26, 2024"
```

Comment:
```suggestion
    "subscription": "Feb 26, 2025"
```

Reply:
Why no timestamp ?

Reply:
I thought it should be more human-readable, and if we do `new Date("Feb 26, 2025")` it will return a timestamp